assets/app/view/game/abilities.rb

```diff
@@ -29,39 +34,64 @@ def render
             store(:show_other_abilities, !@show_other_abilities)
           end
 
-          children << h('button.button', { on: { click: toggle_show } }, @show_other_abilities ? 'Hide' : 'Show')
-          children.concat(render_companies(others)) if @show_other_abilities
+          props = {
+            style: {
+              margin: '0.5rem 0.5rem 0 0',
+              width: '7.3rem',
+            },
+            on: { click: toggle_show },
+          }
+          children << h('button.button', props, @show_other_abilities ? 'Hide Others' : 'Show Others')
+          children.concat(render_companies(others, op_round)) if @show_other_abilities
         end
 
-        if companies.include?(@selected_company)
-          children << h(:div, { style: { margin: '0.5rem 0' } }, @selected_company.desc)
-          children.concat(render_actions)
-        end
+        children << render_selected_company_desc if !op_round && companies.include?(@selected_company)
```

Comment:
hm this looks incorrect, what if you want to exchange a share in the operating round?

the name of this method is bad, it's not just company description, it's also the actions

Reply:
> hm this looks incorrect, what if you want to exchange a share in the operating round?

Dougo works fine:
![image](https://user-images.githubusercontent.com/33390595/88617036-f6f24b00-d095-11ea-98e5-328a9b5a03af.png)
![image](https://user-images.githubusercontent.com/33390595/88617046-fa85d200-d095-11ea-9710-15b3d0374304.png)

Reply:
hm how does it work then? i must not understand the code

Reply:
Maybe. It’s a bit tricky. Abilities rendering is triggered at several places, depending if op_round or not.

Reply:
render_companies does render abilities if op_round
L48 does it if !op_round